count-bottles/action.yml

```
@@ -27,5 +27,5 @@ runs:
   steps:
     - run: ./count.sh '${{ inputs.working-directory }}' '${{ inputs.debug }}'
       working-directory: ${{ github.action_path }}
-      shell: bash
+      shell: /bin/bash -e {0}
```

```suggestion
      shell: /bin/bash -euo pipefail {0}
```

may want to consider that on all of these, assuming it doesn't break them.